src/plugins/wizard/public/application/components/workspace.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiButton,
+  EuiContextMenu,
+  EuiContextMenuPanelItemDescriptor,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiPopover,
+} from '@elastic/eui';
+import React, { FC, useState, useMemo } from 'react';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WizardServices } from '../../types';
+import { useTypedDispatch, useTypedSelector } from '../utils/state_management';
+import { setActiveVisualization } from '../utils/state_management/visualization_slice';
+
+import './workspace.scss';
+
+export const Workspace: FC = ({ children }) => {
+  return (
+    <section className="wizWorkspace">
+      <EuiFlexGroup className="wizCanvasControls">
+        <EuiFlexItem grow={false}>
+          <TypeSelectorPopover />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiPanel className="wizCanvas">
+        {children ? (
+          children
+        ) : (
+          <EuiFlexItem className="wizWorkspace__empty">
+            <EuiEmptyPrompt
+              iconType="visBarVertical"
+              title={<h2>Welcome to the wizard!</h2>}
+              body={<p>Drag some fields onto the panel to visualize some data.</p>}
```

Comment:
Can we have `i18n.translate` for the `title` and `body`?

Reply:
We have a separate issue to track updating all the hard coded string that we ported over from the POC #958